### PR TITLE
[15.0][mig]-hr_worked_days_from_timesheet

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**

### CODE REVIEW

File: hr_worked_days_from_timesheet/views/hr_payslip_view.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                        <button
+                            string="Import from Timesheets"
+                            name="import_worked_days"
+                            type="object"
+                            states="draft"
+                        />
+                    </div>
+                </group>
+            </field>
+            <xpath expr="//sheet/notebook/page[1]/separator[1]" position="replace" />
```
Comment (AaronHForgeFlow) [inline hr_worked_days_from_timesheet/views/hr_payslip_view.xml]: Why is this needed? I mean, why replace instead of invisible? I know this comes form previous versions, but I am not sure why it is like this.

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review

File: hr_worked_days_from_timesheet/views/hr_payslip_view.xml (hunk 1 above)
Comment (JuanyDForgeflow) [reply hr_worked_days_from_timesheet/views/hr_payslip_view.xml]: I'ts invisible now, the replace was just to hide the original separator

Comment (AaronHForgeFlow) [reply hr_worked_days_from_timesheet/views/hr_payslip_view.xml]: ok!

**Review by MiquelRForgeFlow — APPROVED** [review]
